src/components/Search.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+
+type UserType = {
+  id: string;
+  name: string;
+  login: string;
+  imgUrl: string; // avatar_url
+  description: string; // bio
+};
```

Comment:
[제가 올린 PR](https://github.com/Pre-Onboarding-FE-Team07/wanted-codestates-project-7-7-2/pull/2#issue-1170697656) 보시면 아시겠지만 `graphql-codegen`에서 User타입과 같은 응답에 관련된 모든 타입을 자동으로 추가해줍니다. UserType은 추후에 지워야할테고, 합칠 때를 대비해 속성값은 동일하게 맞춰주세요.

```suggestion
type UserType = {
  id: string;
  login: string;
  name?: string | null;
  bio?: string | null;
  avatarUrl: any;
};
```

Reply:
avatarUrl은 왜 any 타입인가요?

Reply:
![image](https://user-images.githubusercontent.com/21158729/158557444-14c8677c-5dbb-48dd-8e92-9aa1ce5effef.png)

`URI`라는 타입이 정의되지 않아서 any로 된 것 같네요. 이 부분은 수정하겠습니다. 타입명만 맞춰주시면 될 것 같습니다!

Reply:
네 알겠습니다!